vizro-core/tests/unit/vizro/models/_components/test_graph.py

```diff
@@ -33,31 +32,6 @@ def standard_px_chart_with_str_dataframe():
     )
 
 
-@pytest.fixture
-def expected_graph():
-    return dcc.Loading(
-        dcc.Graph(
-            id="text_graph",
-            figure=go.Figure(
-                layout={
-                    "paper_bgcolor": "rgba(0,0,0,0)",
-                    "plot_bgcolor": "rgba(0,0,0,0)",
-                    "xaxis": {"visible": False},
-                    "yaxis": {"visible": False},
-                }
-            ),
-            config={
-                "autosizable": True,
-                "frameMargins": 0,
-                "responsive": True,
-            },
-            className="chart_container",
-        ),
-        color="grey",
-        parent_className="loading-container",
-    )
```

Comment:
Have a general question for @antonymilne - is there any preference for whether we put these expected outputs as variables into the relevant unit tests or as fixtures?

Having it directly in the unit tests has the advantage that it's the closest to where it's being used (if only used there). Having it in the fixture makes the unit test easier to overview and people might tend to re-use these fixtures more.

If we do, then I would probably do this consistently throughout the tests here, because currently we mix. I personally prefer it to be consistent, but that's my cleaning up tick 🧹 However, if we don't care, then don't worry about it @nadijagraca  😄

Reply:
I like that! Let me put this into our coding guidelines (which we still have to align on 😄). @nadijagraca - could you then store all of the fixtures in variables (if they are only being used in that unit test)? I think you've done it for the majority already, so good to double-check everywhere (e.g. the slider currently still uses a fixture that is only being used in that unit test) 👍

Reply:
Thank you both for you comments. I'll make sure to follow the guidelines in the future. The reason why I didn't use a variable for slider and range_slider is because component definition is quite long and I was afraid it will clutter the test. But I can move it to make it aligned with other tests.